Index: src/main/java/ru/omsk/neoLab/board/Generators/Calls/Call/ACall.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.omsk.neoLab.board.Generators.Calls.Call;\n\nimport ru.omsk.neoLab.race.ARace;\n\npublic abstract class ACall {\n\n    protected String type;\n\n    public String getType() {\n        return type;\n    }\n\n    public int getMoney(){\n        return 1;\n    }\n\n    //Можно ли забрать территорию\n    public boolean getAbilityCapture(ARace race){\n        return true;\n    }\n\n    //Сколько надо для захвата территории\n    public int getRequirementsForCapture() {\n        return 2;\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/omsk/neoLab/board/Generators/Calls/Call/ACall.java	(revision 9905abadb1bc4179ff47a77805e3c13a99453ec1)
+++ src/main/java/ru/omsk/neoLab/board/Generators/Calls/Call/ACall.java	(date 1594180472930)
@@ -5,6 +5,9 @@
 public abstract class ACall {
 
     protected String type;
+    protected String belongs;
+    protected int countUnits;
+    protected ARace race;
 
     public String getType() {
         return type;
@@ -23,4 +26,20 @@
     public int getRequirementsForCapture() {
         return 2;
     }
+
+    public String getBelongs() {
+        return belongs;
+    }
+
+    public int getCountUnits() {
+        return countUnits;
+    }
+
+    public void setBelongs(String belongs) {
+        this.belongs = belongs;
+    }
+
+    public void setCountUnits(int countUnits) {
+        this.countUnits = countUnits;
+    }
 }
Index: src/main/java/ru/omsk/neoLab/board/Generators/Calls/Call/Earth.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.omsk.neoLab.board.Generators.Calls.Call;\n\npublic class Earth extends ACall {\n\n    public Earth() {\n        type = \"Earth\";\n    }\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/omsk/neoLab/board/Generators/Calls/Call/Earth.java	(revision 9905abadb1bc4179ff47a77805e3c13a99453ec1)
+++ src/main/java/ru/omsk/neoLab/board/Generators/Calls/Call/Earth.java	(date 1594180472935)
@@ -6,4 +6,5 @@
         type = "Earth";
     }
 
+
 }
